protos/camera/camera.proto

```diff
@@ -217,6 +219,14 @@ message FormatStorageResponse {
     CameraResult camera_result = 1;
 }
 
+message SelectCameraResponse {
+    CameraResult camera_result = 1;
+}
+
+message SelectCameraRequest {
+    int32 camera_id = 1;
```

Comment:
This also needs a docstring next to it.

Reply:
Thanks, Updated the commit with the comments,